### PR TITLE
Add package.json file to execute json-align and eslint

### PR DESCRIPTION
**Pull request by madoar**
This PR allows to execute `eslint` and `json-align` through the following commands:

```bash
npm install

npm run lint
npm run json-align
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "name"        : "phoenicis-scripts",
+    "version"     : "1.0.0",
+    "description" : "A collection of Scripts for Phoenicis",
```
Comment (plata) [inline package.json]: scripts should be lower case

Comment (madoar) [reply package.json]: Done

**Comment by plata** [comment]
Should we update the documentation to use `npm run` instead of `eslint` and `json-align` directly?

**Comment by madoar** [comment]
I think it would be a good idea. Especially because the you don't need to pass any parameters

**Comment by plata** [comment]
Can you do it?

**Comment by plata** [comment]
Please wait until we checked the GitHub action before merging this.

**Pull request closed without merging**